cpp/arcticdb/entity/index_range.hpp

```diff
@@ -155,28 +144,44 @@ inline IndexRange universal_range(){ return IndexRange{std::numeric_limits<times
 } //namespace arcticdb::entity
 
 namespace fmt {
-using namespace arcticdb::entity;
 
 template<>
-struct formatter<TimestampRange> {
+struct formatter<arcticdb::entity::TimestampRange> {
     template<typename ParseContext>
     constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
 
     template<typename FormatContext>
-    auto format(const TimestampRange &r, FormatContext &ctx) const {
+    auto format(const arcticdb::entity::TimestampRange &r, FormatContext &ctx) const {
         return fmt::format_to(ctx.out(), "{}-{}", r.first, r.second);
     }
 };
 
 template<>
-struct formatter<IndexRange> {
+struct formatter<arcticdb::entity::IndexRange> {
     template<typename ParseContext>
     constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
 
     template<typename FormatContext>
-    auto format(const IndexRange &r, FormatContext &ctx) const {
+    auto format(const arcticdb::entity::IndexRange& r, FormatContext& ctx) const {
         return fmt::format_to(ctx.out(), "{}-{}", r.start_, r.end_);
     }
 };
 
 } //namespace fmt
+
+namespace arcticdb::entity {
+
+inline bool IndexRange::accept(const IndexValue &index) {
```

Comment:
There is a comment missing about this definition needing the formatter before it's definition (as for the other functions)